plugins/obs-webrtc/whip-output.cpp

```diff
@@ -133,6 +125,13 @@ void WHIPOutput::ConfigureAudioTrack(std::string media_stream_id,
 void WHIPOutput::ConfigureVideoTrack(std::string media_stream_id,
 				     std::string cname)
 {
+	auto encoder = obs_output_get_video_encoder2(output, 0);
+	if (encoder == nullptr) {
+		return;
+	}
+
+	auto codec = obs_encoder_get_codec(encoder);
```

Comment:
Please use `const char *` or `std::string` instead of `auto` here

Reply:
done